integration-cli/requirement/requirement.go

```diff
@@ -27,6 +27,20 @@ func Is(s skipT, requirements ...Test) {
 	}
 }
 
+// IsOneOf checks if the environment satisfies the requirements
+// for the test to run or skips the tests.
+func IsOneOf(s skipT, requirements ...Test) {
+	list := ""
+	for _, r := range requirements {
+		requirementFunc := runtime.FuncForPC(reflect.ValueOf(r).Pointer()).Name()
+		list = list + extractRequirement(requirementFunc) + " "
```

Comment:
Sure

```go

reqs := new(bytes.Buffer)
for _, r := range requirements {
    if r() {
        return
    }
    requirementFunc := runtime.FuncForPC(reflect.ValueOf(r).Pointer()).Name()
    reqs.WriteString(extractRequirement(requirementFunc) + " ")
}
s.Skip("no matched requirement " + reqs.String())
```

Also, since we're duplicating the `runtime.FuncForPC(...)` stuff it might make sense in either `extractRequirement()` or a new function.